### ISSUE TITLE
Allow to customize names for `value` and `label` field names

### ISSUE THREAD

**Issue by KES777** [olifolkerd/tabulator#4641]
*Is your feature request related to a problem? Please describe.**

Usually data is already is loaded via ajax. Database table consists of `id`, `name`, `etc`, `desc`.

**Describe the solution you'd like**
To save processing time and do not generate `values` we could just customize names for `value` and `label` fields:
![image](https://github.com/user-attachments/assets/4784ac95-2110-4884-8918-c6a637858366)

https://tabulator.info/docs/6.3/edit#editor-list
eg. `editorParams: { keyField: 'id', labelField: 'name' }`.
